### DISCUSSION TITLE
5.0.0 Released

### DISCUSSION THREAD

**Discussion by mkleehammer** [mkleehammer/pyodbc#1288] [Announcements]
This is the first Python-3 only release.  Note that macOS binary wheels are still problematic and there is a discussion dedicated to that topic.

The current plan is to test a stable ABI build for 5.1.  This would ensure that the same binary can be used for newer versions of Python such as 5.13.  I have tested this in the past and it is certainly not difficult to build, but I need to ensure performance isn't degraded.  The ABI removes quite a few macros we currently use like GET_ITEM.

**Comment by tweakimp** [comment]
Where can I find the changes that are included in this version?
